src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,59 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <IncludeDllSafeSearchPathAttribute>true</IncludeDllSafeSearchPathAttribute>
+    <NoWarn>$(NoWarn);CS1574;CS3016;CA5379;CA5384</NoWarn>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <RootNamespace></RootNamespace>
+    <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
```

Comment:
Why are you excluding CurrentNetCoreAppFrom package if this is not built for `CurrentNetCoreApp`?

Reply:
I'm not clear on what this library is meant to target.  Is this not going to just be in the netcoreapp shared framework?  We're instead shipping it as a nuget package that can be used with any netstandard2.0 implementation?

Reply:
> We're instead shipping it as a nuget package that can be used with any netstandard2.0 implementation?

Correct.  The plan was to make this as a netstandard2.0 package and try to find some way to avoid consuming it inbox.  (ILMerge, a no-ref private dual-compile, using #define to make all the classes non-public and doing consumption re-compile, etc)

Reply:
In that case @safern comment is correct and you shouldn't have this property as this project isn't building a netcoreapp asset, so just remove this property.